.circleci/config.yml

```diff
@@ -1,28 +1,25 @@
 version: 2.0
 jobs:
   deploy:
-    machine:
-      enabled: true
+    docker:
+      - image: circleci/command-convenience:0.1.658-34232fe
```

Comment:
This should be `circleci/command-convenience:0.1.811-e320c67` or `circleci/command-convenience:0.1` unless you have a good reason to use this older version.

Reply:
no reasons at all, just copied this version from other repo. Thanks for noticing, updated